shared/lib_pv_io_manager.cpp

```diff
@@ -678,6 +678,13 @@ PVSystem_IO::PVSystem_IO(compute_module* cm, std::string cmName, Simulation_IO*
     }
 
     numberOfInverters = cm->as_integer("inverter_count");
+    if (cm->is_assigned("num_inverter_subhourly_clipping")) {
```

Comment:
As annoying as it is, I don't think this should be an input to pvsamv1. I think this value should instead be calculated here if the subhourly clipping model is enabled- you should have sufficient information to do that here, I think?

Reply:
Updated.